src/pages/vanDetails.jsx

```diff
@@ -1,34 +1,49 @@
-import { useParams } from "react-router-dom"
 import { useSelector } from "react-redux";
-import {HiArrowLeft} from "react-icons/hi"
-import { Link } from "react-router-dom";
+import { Link,useParams, useLocation } from "react-router-dom";
 
 function VanDetails() {
   const vanId = useParams();
-  const details = useSelector(prev => prev.vans.value.van)
-  const item =details[vanId.id.slice(0,-1)]
+  const location = useLocation();
+  console.log(location)
+  const details = useSelector((prev) => prev.vans.value.van);
+  const item = details[vanId.id.slice(0, -1)];
```

Comment:
It would be better to use the API endpoint to get a single endpoint so that when a user reloads the page, they can still see the result without having to go back to the vans page.

Reply:
DONE :heavy_check_mark: